src/test/java/com/mael/ttt/players/HumanPlayerTest.java

```diff
@@ -3,27 +3,37 @@
 import com.mael.ttt.Board;
 import com.mael.ttt.ui.SpyConsole;
 import com.mael.ttt.ui.UserInterface;
-import org.junit.Before;
 import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+
+import java.util.Arrays;
+import java.util.Collection;
 
 import static com.mael.ttt.Mark.PLAYER;
 import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
 
+@RunWith(Parameterized.class)
```

Comment:
This is surprising.
I appreciate and welcome that you're thinking about how to keep consistent behaviour between different board sizes. But then again I'd expect to see the parameterised tests for `Board` itself. Not its collaborators.
As mentioned before, a `HumanPlayer` shouldn't have the need to change just because the board size changed, should it?